### PR TITLE
added area

### PR DESCRIPTION
**Pull request by Divsjains6021**

### CODE REVIEW

File: src/main/java/com/propertydekho/infoservice/models/PropFilterableSortableData.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+@Builder
+public class PropFilterableSortableData {
+    @JsonProperty("prop_id")
+    private String propID;
+    @JsonProperty("prop_name")
+    private String propName;
+    @JsonProperty("prop_price")
+    private double propPrice;
+    @JsonProperty("budget_max")
+    private double max_budget;
```
Comment (jainsameer1991) [inline src/main/java/com/propertydekho/infoservice/models/PropFilterableSortableData.java]: @Divsjains6021 Java Variable should be in camel case. Plus see the json variable name(make it max_budget)

File: src/main/java/com/propertydekho/infoservice/models/PropFilterableSortableData.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    public PropFilterableSortableData(){
+
+    }
+
+    public PropFilterableSortableData(String propID, String propName, double propPrice, double max_budget, int bedroom, String saleType, String constructionStatus, String area) {
+        this.propID = propID;
+        this.propName = propName;
+        this.propPrice = propPrice;
+        this.max_budget = max_budget;
```
Comment (jainsameer1991) [inline src/main/java/com/propertydekho/infoservice/models/PropFilterableSortableData.java]: Same comment as above

File: src/main/java/com/propertydekho/infoservice/models/PropFilterableSortableData.java (hunk 1 above)
Comment (Divsjains6021) [reply src/main/java/com/propertydekho/infoservice/models/PropFilterableSortableData.java]: changed in next commit..

File: src/main/java/com/propertydekho/infoservice/models/PropFilterableSortableData.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+@Builder
+public class PropFilterableSortableData {
+    @JsonProperty("prop_id")
+    private String propID;
+    @JsonProperty("prop_name")
+    private String propName;
+    @JsonProperty("prop_price")
+    private double propPrice;
+    @JsonProperty("budget_max")
+    private double budgetMax;
```
Comment (jainsameer1991) [inline src/main/java/com/propertydekho/infoservice/models/PropFilterableSortableData.java]: @Divsjains6021 Please remove budget_max

**Review by jainsameer1991 — APPROVED** [review]

**Pull request merged**